lib/easypost.rb

```diff
@@ -134,7 +134,7 @@ def self.make_request(method, path, api_key = nil, body = nil)
   end
 
   def self.parse_response(status:, body:, json:)
-    if status >= 400
+    if status < 200 || status >= 300
```

Comment:
Nit: We haven't decided what happens with, godforbid, 1xx status codes

Reply:
We don't use em and all the other libs throw an error for them. This change makes this identical to the others.